vmdb/app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -4,13 +4,13 @@ module ContainerGroupHelper::TextualSummary
   #
 
   def textual_group_properties
-    items = %w(namespace name creation_timestamp resource_version restart_policy dns_policy ip)
+    items = %w(name creation_timestamp resource_version restart_policy dns_policy ip)
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
   def textual_group_relationships
     # Order of items should be from parent to child
-    items = %w(ems services containers container_node lives_on)
+    items = %w(ems services containers container_node container_project lives_on)
```

Comment:
Sorry, I meant first of the entities (`ems` excluded):

    ems container_project services containers container_node lives_on